### ISSUE TITLE
Lack of information and improved technology

### ISSUE THREAD

**Issue by gani0325** [visualpython/visualpython#154]
**Is your feature request related to a problem? Please describe.**
1) Lack of detailed explanation
-> We don't know what role each plays when doing subset like loc, iloc ..
![image](https://user-images.githubusercontent.com/76983526/181571465-b61aafc3-72b8-444b-9396-4a0840868785.png)

2) Add-on to snippets
-> I wish autocomplete code format was added
![image](https://user-images.githubusercontent.com/76983526/181571840-f5fa566e-7ebd-4e03-a600-0f13a79bddd4.png)
![image](https://user-images.githubusercontent.com/76983526/181571811-8e9f5e7a-f43a-4d1a-89f8-504466add77d.png)

3) Recommend a model according to the purpose
-> I would like to recommend a model according to the purpose (e.g. LogisticRegression for binary classification)
![image](https://user-images.githubusercontent.com/76983526/181571919-f43a1460-53ba-4d42-92bd-0b25431d7f36.png)

---

**Describe the solution you'd like**
1) Comments or help would make it easier to code
2) It will be possible to complete the code easily, accurately and quickly with a shortcut key or auto-complete code.
3) If you recommend a model suitable for the purpose, good results will be obtained.

**Comment by Minku-Koo** [comment]
yes, I think it's a difficult "word" for python beginner. Suggest to provide help messages.
